project.clj

```diff
@@ -30,6 +30,8 @@
   :scm {:name "git"
         :url "git@github.com:weareswat/stripe-integration.git"}
 
+  :main weareswat.stripe-integration.core
+  :ring {:handler weareswat.stripe-integration.http-component/app}
```

Comment:
Ring is an clojure HTTP server abstraction. https://github.com/ring-clojure/ring